smpp/receiver.go

```diff
@@ -26,7 +26,7 @@ type Receiver struct {
 	TLS                *tls.Config
 	Handler            HandlerFunc
 
-	conn struct {
+	cl struct {
```

Comment:
Any particular reason to rename this from conn to cl? I find conn is more clear.